### DISCUSSION TITLE
Unknow Identifier H

### DISCUSSION THREAD

**Discussion by zhanghonggeng** [mlc-ai/mlc-zh#28] [Q&A]
第三节课卷机的练习中，这里H，W会报错，但是改成8,8之后就可以运行通过了。但是其他的变量没问题。区别好像是下面没有使用H和W，其他都用到了，这是什么问题呀
![PNG图像](https://user-images.githubusercontent.com/43205915/177317467-1a6993f2-b587-49b4-bce9-0a399cd7e51e.png)

**Comment by Hzfengsy** [comment]
感谢提问，H和W需要在TVMScript之前被定义。如果如你所说是因为内部使用的原因，有可能是框架本身实现问题，我们会进一步进行确认
